### PR TITLE
Updated repo readme file

### PR DESCRIPTION
**Pull request by segaz2002**
Added project description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,3 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Browser media player
```
Comment (segaz2002) [inline README.md]: Can you make it bigger please ?

File: README.md [hunk 2]
```diff
@@ -1,68 +1,3 @@
...
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+React app to play music/media from browser local storage.
```
Comment (segaz2002) [inline README.md:3]: Please add more comments

Comment (sleekyom) [inline README.md:3]: Please describe your comment properly

**Review by sleekyom — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (segaz2002) [reply README.md:3]: fixed.

**Review by sleekyom — APPROVED** [review]

**Pull request merged**